### PR TITLE
1. Fixed user being able to submit transportation claim through "Claims"

### PR DESCRIPTION
**Pull request by CoreVisional**
2. Update .gitignore to ignore vscode files

### CODE REVIEW

File: Payroll.Web/Controllers/ClaimsController.cs [hunk 1]
```diff
@@ -67,7 +67,9 @@ public async Task<IActionResult> Create([FromForm] Claim claim)
             ModelState.Remove("DeletedTime");
             ModelState.Remove("User");
 
-            if (ModelState.IsValid && claim.ClaimType != ClaimType.Work && claim.File != null)
+            var claimTypeValues = Enum.GetValues(typeof(ClaimType));
+
+            if (!ModelState.IsValid && claim.ClaimType.Equals(claimTypeValues) && claim.File != null)
```
Comment (ErwinSuwito) [inline Payroll.Web/Controllers/ClaimsController.cs:72]: `claim.ClaimType.Equals(claimTypeValues)` will always return false as you're comparing an enum with an array of enum. A better way is to use switch to check which claim type is allowed to be submitted through this form. (Though there are none at the moment)

**Review by ErwinSuwito — CHANGES_REQUESTED** [review]

File: Payroll.Web/Controllers/ClaimsController.cs (hunk 1 above)
Comment (CoreVisional) [reply Payroll.Web/Controllers/ClaimsController.cs:72]: Refactored the code to use switch-case for claim.ClaimType check. Thanks for the clarification regarding the returning value for the equality check against an enum with an array of enum also.

**Review by CoreVisional — COMMENTED** [review]
Please kindly disregard commit "db119a" as it showed that I already committed the claim.ClaimType changes only to discover upon restarting my VS that it committed the incorrect changes.

File: Payroll.Web/Controllers/ClaimsController.cs (hunk 1 above)
Comment (ErwinSuwito) [reply Payroll.Web/Controllers/ClaimsController.cs:72]: What I meant is to remove `claim.ClaimType.Equals(claimTypeValues)` from the if conditions, then put the switch inside the if body

Comment (CoreVisional) [reply Payroll.Web/Controllers/ClaimsController.cs]: Updated the if body with the switch case statement on commit "f054d57".

Comment (CoreVisional) [reply Payroll.Web/Controllers/ClaimsController.cs]: Updated claim.ClaimType check on commit "06ed64b".

**Review by ErwinSuwito — APPROVED** [review]

**Pull request merged**